### DISCUSSION TITLE
Make server public

### DISCUSSION THREAD

**Discussion by westpfelia** [Nimdy/Dedicated_Valheim_Server_Script#41] [Q&A]
Hey I was wondering whats the way to make the server public? I know the docker container I was trying to make work had a "public 1" switch.

**Comment by Nimdy** [comment]
public 1 was killed off day one. Not required.  IronGate public listing is hit or miss.... its a Steam and IronGate API issue.

**Comment by westpfelia** [comment]
Oh, well that makes sense! Connecting via IP it is then! 


Thanks

**Comment by Nimdy** [comment]
GLHF!
